modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/form_builder.jspf

```diff
@@ -57,13 +57,13 @@ if (Validator.isNotNull(script)) {
 					<div class="tabbable">
 						<div class="tabbable-content">
 							<ul class="lfr-nav nav nav-tabs nav-tabs-default">
```

Comment:
We could probably remove `nav-tabs-default`.
I still see this still being used in:
- modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/asset/full_content.jsp
- modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/ddm/ddm_form_builder.jsp
- modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/item_viewer.js
- modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/edit_kaleo_definition_version.jsp
- modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-web/src/main/resources/META-INF/resources/admin/process/workflow.jsp